docs/components/Autocomplete/WebV2.stories.tsx

```diff
@@ -302,23 +304,31 @@ const TemplateHeaderFooter: ComponentStory<typeof Autocomplete> = () => {
   return (
     <Content>
       <Heading level={4}>Persistent header/footer actions</Heading>
+      <input type="text" />
       <Autocomplete
         version={2}
         placeholder="Search"
         value={value}
+        openOnFocus={false}
         onChange={setValue}
+        onBlur={() => console.log("blurred")}
         inputValue={inputValue}
         onInputChange={setInputValue}
         menu={defineMenu<OptionLike>([
-          { type: "header", label: "Pinned header", shouldClose: false },
+          {
+            type: "header",
+            label: "Pinned header",
+            shouldClose: false,
+            onClick: () => console.log("clicked"),
```

Comment:
this has been missing for a while, we have `shouldClose` but no onClick

so I've added that and made the footer text only